### ISSUE TITLE
Constants defined in a Module in an ActiveX library are not visible in tthe consumer project

### ISSUE THREAD

**Issue by FullValueRider** [WaynePhillipsEA/twinbasic#519]
**Describe the bug**
In my TwinLib64 ActiveX library I have a module (Char) that defines some public string constants.  These constants are not visible in the project that references the library.  In the snapshot below I've included a copy of the Char Module as a comment.

![NoChars](https://user-images.githubusercontent.com/81257042/144376171-291d0f65-31c7-4f8a-bb71-8ffa95a0c51c.jpg)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - twinBASIC compiler version 0.13.13

**Comment by WaynePhillipsEA** [comment]
VB6 doesn't expose anything from a standard module either, so this needs some thought about how best to allow exposure of them without changing the expected behaviour for VB6 projects.

**Comment by bclothier** [comment]
I agree with Kr00l that attributes are the best way to handle this. We shouldn't need to extend the syntax for this and there may be additional considerations that would be difficult to manage were it a syntax vs. it being an attribute.

**Comment by WaynePhillipsEA** [comment]
Yup, this should be controlled via attributes. I'm open to suggestions with regards the attributes syntax for this...

**Comment by Kr00l** [comment]
`[ ModuleCOMExposed(True) ]`

Though this should be meaningful for `Public` members only.

**Comment by bclothier** [comment]
I think simply `[ Exposed(True) ]` will suffice.

However, considering the discussion in #564 where we might float the idea of an `[ Instancing(...)]` to provide a textual in-source representation of the property to consolidate the 6 different attributes, this might have some ramifications. As noted there, there are fewer instancing choices than possible combinations of the 6 attributes so I assume other combinations are invalid or otherwise is ignored. We want to avoid exposing an impossible combination which is why I think an `Instancing` attribute in place of 6 legacy attributes would be a good solution. Not sure if that has bearing on the members that we want to include in the generated typelibs, however.
